src/models/sequence/sequence.go

```diff
@@ -8,28 +8,33 @@ import (
 	"strings"
 )
 
-//Validatetheinput validates the input data and ensures that it doesn't contain any separators, symbols etc
+//ValidateTheInput validates the input data and ensures that it doesn't contain any separators, symbols etc
 //that may endanger the correct work of the algorithm
-
-func Validatetheinput(lenghtOfSequence, minSquare string,r *regexp.Regexp) (lenghtOfSequence1, minsquare1 int, err error) {
+func ValidateTheInput(lenghtOfSequence, minSquare string,r *regexp.Regexp) (lenghtOfSequence1, minSquare1 int, err error) {
```

Comment:
1) do you still use `string` for the input parameters?
2) pls, rename them to `inputLength` - `outputLength` and `inputMinSquare` - `outputMinSquare`

Reply:
agree

Reply:
Changed.